Package.swift

```diff
@@ -19,6 +19,7 @@ func resolveDependencies() -> [Package.Dependency] {
 
 func resolveTargets() -> [Target] {
     let objcSources = ["Purchases/Info.plist",
+                       "Purchases/Attribution",
```

Comment:
we might eventually wanna get fancier here and just list the directory so that we don't have to add these one by one, but this PR got bigger than expected as is